twoliter/embedded/Makefile.toml

```diff
@@ -329,11 +329,10 @@ dependencies = ["setup"]
 script_runner = "bash"
 script = [
 '''
-if [ -f "${BUILDSYS_SOURCES_DIR}/Cargo.toml" ]; then
-  cargo fetch --locked --manifest-path "${BUILDSYS_SOURCES_DIR}/Cargo.toml"
-fi
-
-cargo fetch --locked --manifest-path "${BUILDSYS_ROOT_DIR}/Cargo.toml"
```

Comment:
We still need this or else we are breaking the ability to go to one workspace. Are you saying we will no longer have one workspace as a possibility? It would need to be something like:

```bash
if [ -f "${BUILDSYS_ROOT_DIR}/Cargo.toml" ]; then
  cargo fetch --locked --manifest-path "${BUILDSYS_ROOT_DIR}/Cargo.toml"
fi
```